frontend/src/components/mine/Variances/MineVarianceTable.js

```diff
@@ -32,6 +32,8 @@ const defaultProps = {
 export class MineVarianceTable extends Component {
   errorStyle = (isOverdue) => (isOverdue ? { color: errorRed } : {});
 
+  hideColumn = (isApplication) => (isApplication ? "column-hide" : "");
```

Comment:
Since this is a pure function, it doesn't need to be a method on the class (ie. it doesn't need to be this-aware). For simplicity, I'd suggest moving it out of the class. The same can be said for `errorStyle`